### PR TITLE
Mobile single level sidenav

### PR DESCRIPTION
**Pull request by kuramochi-coder**
Added mobile view for singe level sidenav

Mobile view of sidenav and title:
image

Mobile sidenav selection view (updated):
image

### CODE REVIEW

**Review by teochengyong — CHANGES_REQUESTED** [review]
Another question, currently we have 2 layouts. 
- Layout-page and Layout-sidenav. Would it be confusing for layout-page to have the sidenav as well?

File: _layouts/layout-page.html [hunk 1]
```diff
@@ -3,10 +3,29 @@
...
+              <span class="has-text-weight-normal">{{ page_name | split: "-" | join: " " | capitalizewords }}</span>
+              <span class="sgds-icon sgds-icon-chevron-down"></span>
+          </button>
+          <ul class="mobile-sidenav-menu is-hidden">
+              {% include sidenav-single-level.html %}
+          </ul>
+      </div>
+    </div>
+</section>
+{% endif %}
```
[the remark this answers is not in the document]
Comment (kuramochi-coder) [reply _layouts/layout-page.html:26]: Currently layout-default also contains a mobile multi-level sidenav which appears at the very top. In order to position the single-level mobile sidenav in the middle after the page headers, it had to be placed in layout-page. 

Will change the current mobile single-level sidenav to a partial file.

**Review by teochengyong — COMMENTED** [review]
Mostly looks good and just minor comment

**Review by teochengyong — APPROVED** [review]
LGTM  🙂

**Pull request merged**